fortimanager.json

```diff
@@ -0,0 +1,1193 @@
+{
+    "appid": "75d83a2e-e7f7-43b8-9378-5ea7d4a7e972",
+    "name": "FortiManager",
+    "description": "FortiManager",
+    "type": "network security",
+    "product_vendor": "Fortinet",
+    "logo": "logo_fortinet.svg",
+    "logo_dark": "logo_fortinet_dark.svg",
+    "product_name": "FortiManager",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) Splunk, 2023",
+    "app_version": "1.0.0",
+    "utctime_updated": "2023-08-03T19:40:27.182156Z",
+    "package_name": "phantom_fortimanager",
+    "main_module": "fortimanager_connector.py",
+    "min_phantom_version": "6.0.0",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "url": {
+            "description": "Base URL of FortiManager instance",
+            "data_type": "string",
+            "required": true,
+            "order": 0
+        },
+        "user": {
```

Comment:
the asset uses `username` instead of `user` so you might get a failure when you try to run tests

Reply:
changed